### PR TITLE
Add 2021 Tools Plenary slides for Project Jupyter

### PR DESCRIPTION
**Pull request by isabela-pf**
Submitting slides for Project Jupyter's update at the Tools Plenary. Thanks for organizing a lovely conference!

@carreau I didn't know if you had an orcid to fill in on `info.json`. If so, please feel free to add it.

### CODE REVIEW

**Comment by scoobies** [comment]
Hi!, I'm @scoobies, a friendly bot.

Type ```@scoobies help``` to discover how I can help you.

File: presentations/tools/project-jupyter/info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "authors": [
+        {
+            "name": "Isabela Presedo-Floyd",
+            "affiliation": "Project Jupyter",
+            "orcid": ""
+        },
+        {
+            "name": "Matthias Bussonnier",
+            "affiliation": "Project Jupyter",
+            "orcid": ""
```
Comment (Carreau) [inline presentations/tools/project-jupyter/info.json]:
```suggestion
            "orcid": "0000-0002-7636-8632"
```

**Comment by Carreau** [comment]
> @Carreau I didn't know if you had an orcid to fill in on `info.json`. If so, please feel free to add it.

Thanks, done. I'll add a comment on the discourse thread to point to here for other contributors.

**Comment by deniederhut** [comment]
> Thanks for organizing a lovely conference!

We're so glad you could come!

**Pull request merged**